cli/tests/e2e/snapshots/test_metrics/test_metrics_payload/metrics-payload.json

```diff
@@ -3,6 +3,7 @@
   "environment": {
     "ci": null,
     "configNamesHash": "d3e03c9938537c0a0668cec7204f37f1e9da87d03e18e8cc444dd7454b36c4d5",
+    "engineType": "OSS",
```

Comment:
why is this engineType not engineRequested?

Reply:
i have the same question. this is weird, wonder why it's there

Reply:
Oh my bad that file shouldn't be there